web/src/compose_setup.js

```diff
@@ -508,6 +509,13 @@ export function initialize() {
         }
     });
 
+    $(".compose-scrollable-buttons").on(
+        "scroll",
+        _.throttle((e) => {
+            compose_ui.handle_scrolling_formatting_buttons(e);
+        }, 150),
+    );
```

Comment:
What's the motivation for throttling here? I did feel when playing with it that I felt a bit of lag when asking it to scroll, and I'm wondering if there's something we should tune here or elsewhere.

Reply:
The throttling here is to make sure that we're not trying to update the formatting buttons every time the `scroll` event fires (and it fires a whole lot, and definitely causes unnecessary lag without throttling). But shouldn't be related to lag in scrolling when you click the scrolling buttons, though.

I think the issue with `compose_ui` is that it doesn't have an initialize event, and I wanted to register the handler for the scroll event on the buttons adjacent other event listeners on compose, which seem to mostly be in the `compose_setup` file.

Reply:
OK, that seems reasonable.